lib/static/components/favicon-status.js

```diff
@@ -0,0 +1,76 @@
+'use-strict';
+
+import {PureComponent} from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+
+const DEFAULT = 'default';
+const SUCCESS = 'success';
+const FAILED = 'fail';
+const RUNNING = 'running';
+
+class FaviconStatus extends PureComponent {
+    static propTypes = {
+        status: PropTypes.string
+    }
+
+    link = null;
+
+    getFaviconURL(color) {
+        const canvas = document.createElement('canvas');
+        const diameter = 32;
```

Comment:
maybe we should get ability to control this value from plugin config?

Reply:
Not sure that will be useful for configuration, can you imagine case when you need configure size of favicon?

Reply:
Perhaps you're right, let's at least put it into constants